Added tmin, tmax arguments to gtltcube_mp() call.

SeanCGriffin opened the pull request.

SeanCGriffin committed:
Added tmin, tmax arguments to gtltcube_mp() call.

kialio commented:
Hi, 

The tmin and tmax arguments are there in case you want to run gtltcube_mp without an event file.  In this case, I think we'll always have an event file so it's better to just set both of them to '0' which will default to the time in the event file.

Jeremy

SeanCGriffin committed:
Change gtltcube_mp() time arguments

Pass tmin = 0, tmax = 0 to gtltcube_mp so that it defaults to the event file times.

SeanCGriffin commented:
Done.

On Tue, Feb 4, 2014 at 2:05 PM, Jeremy Perkins notifications@github.comwrote:

> Hi,
> 
> The tmin and tmax arguments are there in case you want to run gtltcube_mp
> without an event file. In this case, I think we'll always have an event
> file so it's better to just set both of them to '0' which will default to
> the time in the event file.
> 
> Jeremy
> 
> ## 
> 
> Reply to this email directly or view it on GitHubhttps://github.com/kialio/LATAnalysisScripts/pull/8#issuecomment-34094688
> .

## 

-=//\=-
